Neos.Flow/Classes/Mvc/Controller/Arguments.php

```diff
@@ -18,6 +18,7 @@
  * A composite of controller arguments
  *
  * @api
+ * @extends \ArrayObject<Argument>
```

Comment:
Doesn't change the typehinting in IDE (yet), but psalm should be happy about this.